hydrus/app.py

```diff
@@ -535,8 +761,10 @@ def app_factory(API_NAME: str="api") -> Flask:
                      "/<string:path>", endpoint="item_collection")
     api.add_resource(Item, "/" + API_NAME +
                      "/<string:path>/<int:id_>", endpoint="item")
-    api.add_resource(Items,"/"+ API_NAME +
-                     "/<string:path>/list",endpoint="items")
+    api.add_resource(Items, "/" + API_NAME +
+                     "/<string:path>/add/<int_list>", "/" + API_NAME +

```

Comment:
I believe it should be `/get` instead of `/add`.

Reply:
but its being used to add objects right ?

Reply:
I haven't run it yet.